src/AppInstallerCLICore/Commands/InstallCommand.cpp

```diff
@@ -137,6 +139,11 @@ namespace AppInstaller::CLI
                 throw CommandException(Resource::String::InvalidArgumentValueErrorWithoutValidValues, Argument::ForType(Args::Type::Locale).Name(), {});
             }
         }
+
+        if (execArgs.Contains(Execution::Args::Type::DisplayNotes) && execArgs.Contains(Execution::Args::Type::SuppressNotes))

```

Comment:
Do we really need two arguments for this? I know for sure in PS we would do it as a single switch to which you could pass `$true` or `$false` to specify, but I'm not sure here.

Reply:
Repeating myself, but I don't think we need any arguments for this.

Reply:
Removed arguments